src/master/develop/history.md

```diff
@@ -48,8 +48,39 @@ Note that, similar with many other methods, `history` method is overloaded to al
 :::
 
 ::: tab Python
-This feature is not yet supported or not documented.
-Do you want to make a feature request or help out? Open an issue on [Python sdk github project](https://github.com/codenotary/immudb-py/issues/new)
+
+Python immudb sdk currently doesn't support `SinceTx` parameter
```

Comment:
Please add a github issue to fix that.

Reply:
https://github.com/codenotary/immudb-py/issues/37